crates/core/src/db/commit_log.rs

```diff
@@ -122,8 +122,8 @@ impl CommitLog {
                     // Increment rows inserted metric
                     let metric = rows_inserted.with_label_values(
                         &ctx.txn_type(),
-                        &ctx.database_id(),
-                        &ctx.reducer_id().unwrap_or(0),
+                        &ctx.database(),
+                        ctx.reducer_name().unwrap_or(""),
```

Comment:
I'm fine if this also changes metrics to track reducer names rather than ids. But if so, please include that in the commit message, and assure me that it's done uniformly for all metrics.